serveradmin/settings.py

```diff
@@ -54,6 +54,7 @@
         },
     },
 }
+DATABASE_ROUTERS = ['serveradmin.powerdns.routers.PowerDNSRouter']
```

Comment:
Weren't there more Routers already? Thought, I remember that we already added some routers before. But maybe I misremembering something here.

Reply:
No there are no more routers currently. Very early back in the days there may have been some when we were switching from MySQL to PostgreSQL but I am not sure.

Reply:
Ah, possible. Nevermind then.